persistence/sql/persister_oauth2.go

```diff
@@ -108,6 +104,31 @@ func (p *Persister) sqlSchemaFromRequest(rawSignature string, r fosite.Requester
 	}, nil
 }
 
+func (p *Persister) marshalSession(session fosite.Session) ([]byte, error) {
+	sessionBytes, err := json.Marshal(session)
+	if err != nil {
+		return nil, err
+	}
+
+	if sessionBytes, err = p.maybeEncryptSession(sessionBytes); err != nil {
```

Comment:
Could you make this explicit? A simple `:=` here would disable encryption, a significant security feature. Thanks!